src/Popper.js

```diff
@@ -116,8 +116,8 @@ export class InnerPopper extends React.Component<PopperProps, PopperState> {
     !this.popperNode || !this.state.data
       ? initialStyle
       : {
-          position: this.state.data.offsets.popper.position,
           ...this.state.data.styles,
+          position: this.state.data.offsets.popper.position,
```

Comment:
Isn't `data.offsets.popper.position` an exact type? What error do we get if we keep it where it was before?

Reply:
The actual issue is with the type of `this.state.data.styles`. Its Flow type is `CSSStyleDeclaration` (from the CSSOM Flow types), which happen to have `number` as keys. 

For some reason, this makes Flow unsure about whether `position` would be overwritten by the spread or not. Moving `position` after the spread removes ambiguity.

Here is the original Flow error :

```
Error --------------------------------------------------------------------------------------------- src/Popper.js:120:12

Cannot determine a type for object literal [1]. `CSSStyleDeclaration` [2] cannot be spread because the indexer
number [3] may overwrite properties with explicit keys in a way that Flow cannot track. Can you spread
`CSSStyleDeclaration` [2] first or remove the indexer?

   src/Popper.js:120:12
   120|         ...this.state.data.styles,
                   ^^^^^^^^^^^^^^^^^^^^^^

References:
   src/Popper.js:118:9
                v
   118|       : {
   119|           position: this.state.data.offsets.popper.position,
   120|         ...this.state.data.styles,
   121|         };
                ^ [1]
   node_modules/popper.js/index.js.flow:39:13
    39|     styles: CSSStyleDeclaration,
                    ^^^^^^^^^^^^^^^^^^^ [2]
   ../../AppData/Local/Temp/flow/flowlib_218c75f5/cssom.js:408:11
   408|   [index: number]: string;
                  ^^^^^^ [3]
```

Reply:
@FezVrasta, is the change OK for you ?